clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -192,24 +205,19 @@ def complete_clinical_data(
     df_clinical_complete = df_imaging.merge(
         df_demographics, how="inner", on=merge_key
     ).drop(columns="diagnosis")
-    df_clinical = df_clinical.dropna(subset=merge_key)
-    return df_clinical_complete.merge(
-        df_clinical[
-            [
-                "blinded_code",
```

Comment:
Might be a stupid question, but you don't need to filter these anymore ?

Reply:
Nop I don't need to. I think it was never needed in the first place, since we filter afterwards. It just avoided having to deal with the entire data and the problems it may contain